### PR TITLE
Search apps by abbreviation

### PR DESCRIPTION
**Pull request by KELiON**
Search apps by abbreviation, i.. 
 *   League of Legends -> lol
 *   AudioBookBinder -> abb

Implemented #65

### CODE REVIEW

File: app/main/plugins/core/basic-apps/linux.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import path from 'path'
 import fs from 'fs'
 import uniq from 'lodash/uniq'
 import flatten from 'lodash/flatten'
+import getAbbr from 'lib/getAbbr'
```
Comment (Kageetai) [inline app/main/plugins/core/basic-apps/linux.js]: where is that coming from? Can't find that file anywhere?!

File: test/plugins/osx-apps/index.spec.js [hunk 2]
```diff
@@ -12,6 +12,16 @@ const appsList = [
...
+  },
+  {
+    name: 'League of Legends',
+    filename: 'League of Legends.app',
+    path: '/Applications/League of Legends.app'
+  },
+  {
+    name: 'AudioBookBinder',
+    filename: 'AudioBookBinder.app',
+    path: '/Applications/AudioBookBinder.app'
```
Comment (Kageetai) [inline test/plugins/osx-apps/index.spec.js]: Do all abbreviations have to be hardcoded?

File: app/main/plugins/core/basic-apps/linux.js (hunk 1 above)
Comment (KELiON) [inline app/main/plugins/core/basic-apps/linux.js]: @Kageetai it is in `lib/getAbbr`, I forgot to push this file initially)

File: test/plugins/osx-apps/index.spec.js (hunk 2 above)
Comment (KELiON) [inline test/plugins/osx-apps/index.spec.js]: @Kageetai what do you mean?

Comment (Kageetai) [inline test/plugins/osx-apps/index.spec.js]: Okay, as I didn't see the file `getAbbr` before I did not know how you get the abbrevations ;)
Oh and I overlooked that this is just the test file X)

**Pull request merged**